### PR TITLE
Added scripts for 1-5-advocacy-ca email

### PR DESCRIPTION
**Pull request by aoby**

### CODE REVIEW

File: bin/mailing-1-5-advocacy-ca/generate-ca-list [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+# To the below lists targeted to the state of CA:
+
+# 1. all petition signers
+query_subscribed_contacts(q: 'kind_s:Petition && (state_code_s:ca || create_ip_state_s:California)')
+
+# 2. Code Studio teacher accounts
+query_subscribed_contacts(q: 'kind_s:user && role_s:teacher && create_ip_state_s:California')
+
+# 3. Hour of Code organizers (2014, 2015)
```
Comment (laurelfan) [inline bin/mailing-1-5-advocacy-ca/generate-ca-list:13]: we don't want 2013?

Comment (aoby) [inline bin/mailing-1-5-advocacy-ca/generate-ca-list:13]: That was the requirement. I'm confirming with Roxanne...

Comment (aoby) [inline bin/mailing-1-5-advocacy-ca/generate-ca-list:13]: confirmed.

**Comment by laurelfan** [comment]
cool, lgtm

**Pull request merged**